CHANGES.rst

```diff
@@ -86,16 +86,16 @@ dark_current
 datamodels
 ----------
 
+- Added a new info method, similar to the method in astropy fits [#2268]
+
 - The ``DataModel`` ``__hasattr__`` method has been replaced by ``hasattr``.
   The former created the attribute when it was accessed. [#2275]
 
 - Improved error messaging when loading fits files into data models. [#2298]
 
 - New warning message when opening a file without DATAMODL keyword. [#2248]
 
-- New info method, similar to the method in astropy fits [#2268]
```

Comment:
Did you intend to delete this without replacing it?

Reply:
It's still there, I just moved it to the top of the list